configserver/src/main/java/com/yahoo/vespa/config/server/session/SessionCache.java

```diff
@@ -16,9 +16,7 @@
     private final HashMap<Long, SESSIONTYPE> sessions = new HashMap<>();
 
     public synchronized void addSession(SESSIONTYPE session) {
-        if (sessions.containsKey(session.getSessionId()))
-            throw new IllegalArgumentException("There already exists a session with id '" + session.getSessionId() + "'");
-        sessions.put(session.getSessionId(), session);
+        sessions.putIfAbsent(session.getSessionId(), session);
```

Comment:
Intentional that already existing session no longer causes an exception ?

Reply:
Yes, I could not find any issues with changing this